app/src/main/java/org/schabi/newpipe/util/StreamDialogDefaultEntry.java

```diff
@@ -0,0 +1,165 @@
+package org.schabi.newpipe.util;
```

Comment:
I'm not sure if this class should belong into util.

Reply:
I moved all classes related to the InfoItemDialog into an own package.